### PR TITLE
demeteorizer - now with arbitrary JSON enhancement

### PR DESCRIPTION
**Pull request by robertBrown**
usesage:

demeteorizer -l '{"xx":"xx","yy":"yy","zz":"zz"}'

### CODE REVIEW

**Comment by fiveisprime** [comment]
You'll need to rebase master onto your branch before I can
review and merge this. :)

File: lib/cli.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ program
   .version(require('../package.json').version)
   .option('-o, --output <path>', 'Output folder for converted application. Defaults to ./.demeteorized.')
   .option('-r, --release <version>', 'The Meteor version. Defaults to latest installed.')
+  .option('-l, --arbitrary <\'{"xx":"xx"}\'> ', 'Add arbitrary JSON to package.json')
```
Comment (fiveisprime) [inline lib/cli.js]: I like the option to be `--json` or `-j`. Arbitrary is way too much to type out in a terminal

Comment (robertBrown) [inline lib/cli.js]: Ok, will do. :)

On Thursday, April 2, 2015, Matt Hernandez <notifications@github.com> wrote:

> In lib/cli.js
> <https://github.com/onmodulus/demeteorizer/pull/119#discussion_r27661600>:
>
> > @@ -6,6 +6,7 @@ program
> >    .version(require('../package.json').version)
> >    .option('-o, --output <path>', 'Output folder for converted application. Defaults to ./.demeteorized.')
> >    .option('-r, --release <version>', 'The Meteor version. Defaults to latest installed.')
> > +  .option('-l, --arbitrary <\'{"xx":"xx"}\'> ', 'Add arbitrary JSON to package.json')
>
> I like the option to be --json or -j. Arbitrary is way too much to type
> out in a terminal
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/onmodulus/demeteorizer/pull/119/files#r27661600>.
>


-- 


- Robert Brown

**Comment by robertBrown** [comment]
whoo hoo!!



- Robert Brown

**Pull request closed without merging**